pkgs/development/tools/jbang/default.nix

```diff
@@ -14,7 +14,7 @@ stdenv.mkDerivation rec {
   installPhase = ''
     runHook preInstall
     rm bin/jbang.{cmd,ps1}
-    rmdir tmp
+    rm -rf tmp
```

Comment:
Is there a reason not just update to the latest version and than drop this line?

Reply:
if someone want than an older version, they could just put the rm back in an override.

Reply:
It's being defensive. Why risk breaking a perfectly reasonable override? If I wanted an older version I'd have to copy the whole `installPhase` from the base derivation (just like I have to no if I want a new version), and that seems like unnecessary toil.

Reply:
But I don't care TBH. See latest revision.